sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -601,7 +572,7 @@ class ExistenceJoinIterator(
    * the value "false", scattering "true" into column FC will produce the "exists"
    * column of ExistenceJoin
    */
```

Comment:
the comment should be moved to the abstract parent class method declaration

Reply:
done